src/test/java/core/basesyntax/services/impl/FileWriterTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.services.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.services.FileWriter;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileWriterTest {
+    private static final String PATH_TO_TEST_FILE = "src/resources/testReport.txt";
+    private static final String EXPECTED_REPORT = "fruit,quantity, banana,152, apple,90";
+    private static final File TEST_FILE = new File(PATH_TO_TEST_FILE);
+    private FileWriter fileWriter;
+
+    @Before
+    public void createFiles() throws Exception {
```

Comment:
U can split this into two methods, if u need create files before every test.
new FileWriterImpl its setup method with annotation BeforeClass.